bin/pt-mysql-summary

```diff
@@ -2162,41 +2150,6 @@ parse_wsrep_provider_options () {
    ' "$looking_for"
 }
 
-report_jemalloc_enabled() {
```

Comment:
why this has been removed?

Reply:
Seems like an error in a merge as I did not change that. I'll see if I can find when that happened, or will submit a new PR with only the have_innodb changes on it.

Reply:
The problem is introduced when I run update-modules for pt-mysql-summary, so I'll close this PR, create a new one changing just the module, and let you check if my change is causing that to break (I could not find other references to have_innodb but I could be wrong) or if you can update pt-mysql-summary yourself based on the updated module. Thanks!